client/app/helper/url-helper.service.js

```diff
@@ -0,0 +1,12 @@
+(() => {
+
+    class UrlHelper {
```

Comment:
I'm guessing this item is commonly used in the API?
Sometimes you call this function using 3 parameters.

Reply:
yes but as now its only called with 2 parameters